### DISCUSSION TITLE
Help with Organizations/Spaces

### DISCUSSION THREAD

**Discussion by Th4ntis** [GitbookIO/community#318] [0. General bugs and questions] [answered]
I started Gitbook for taking notes and sharing it, I made a "Personal" section. Well I was curious on the upgraded Pro features so I can maybe import my notes from another note taking software via PDF or copy/paste. This had me make an organization. Now everything I had on my "Personal" section, got moved to into the organization by accident. Is there any way I can move it back to my "Personal"?

I'm trying to move the "CyberSec" space from the "Team Library", back to the "Personal library" under it's own space. Please tell this is possible.
![librewolf_JSBpQILLXU](https://github.com/GitbookIO/community/assets/53808039/000cbb2a-e6cb-440b-a336-a1b636406e70)
![librewolf_N4q8sgcXU2](https://github.com/GitbookIO/community/assets/53808039/24d2dadf-b40e-4281-839a-a18d500ecc49)

**Comment by aikoii** [comment, marked as answer]
Hey @Th4ntis! An organization on the free Personal plan is similar to the Personal library, but with additional features and more flexibility. We are migrating all content currently in a Personal library into a new organization to bring these features to _all_ users (or you can move spaces yourself). So you won't be able to move your "CyberSec" space back to the Personal library (we are not supporting most features there anyway).

By the way, if you're on a free plan, you can still import your notes via our [Import feature](https://docs.gitbook.com/content-creation/import) or copy/paste without any issues 🙂.
